units/steelhive/steel_fideliant.cfg

```diff
@@ -15,7 +15,7 @@
     {AMLA_DEFAULT}
     cost=44
     usage=fighter
-    # wmllint: local spelling Fideliants
+	# wmllint: local spelling Fideliants
```

Comment:
what happened here? this spacing showed up out of nowhere.

Reply:
No idea. I tried with tab and no tab and the result is always a difference wrt the original